server/db/connection.js

```diff
@@ -1,11 +1,17 @@
 const { Client } = require('pg');
-const dotenv = require('dotenv');
+require('dotenv').config();
 
 const client = new Client({
   user: process.env.USER,
-  port: process.env.PORT,
+  host: process.env.HOST,
+  port: process.env.PGPORT,
```

Comment:
you do have a DB specific port you've named in your `.env` file! so maybe my comment above on your `example.env` file isn't relevant? I don't see the `PGPORT` in the `example.env`, but maybe I'm missing it!

Reply:
Good catch! I'll add PGPORT